wger/exercises/views/exercises.py

```diff
@@ -54,13 +54,22 @@ class ExerciseListView(ListView):
     template_name = 'exercise/overview.html'
     context_object_name = 'exercises'
 
+    def dispatch(self, *args, **kwargs):
+        self.filter_by = self.request.GET.get('filter_by', None)
+        try:
+            self.language = Language.objects.get(short_name=self.filter_by)
+        except Language.DoesNotExist:
+            self.language = None
+        return super().dispatch(*args, **kwargs)
+
     def get_queryset(self):
         """
         Filter to only active exercises in the configured languages
         """
-        languages = load_item_languages(LanguageConfig.SHOW_ITEM_EXERCISES)
-        return Exercise.objects.accepted() \
-            .filter(language__in=languages) \
+        # TODO WRITE LOGIC FOR EXERCISES
```

Comment:
Could you please explain what's the reason behind this todo comment, `"WRITE LOGIC FOR EXERCISES"` seems too abstract.

Reply:
Sorry I already did that 
Must have forgotten to remove the comment
Fixin it Asap